### PR TITLE
Merge 2.9.x to master 

### PR DESCRIPTION
**Pull request by Evangelink**
Overrides #4674

### CODE REVIEW

**Comment by Evangelink** [comment]
@dotpaul for review

**Comment by mavasani** [comment]
Thanks @Evangelink!

File: src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md [hunk 1]
```diff
@@ -1 +1,5 @@
-; Please do not edit this file manually, it should only be updated through code fix application.
\ No newline at end of file
+; Please do not edit this file manually, it should only be updated through code fix application.
+### Changed Rules
+Rule ID | New Category | New Severity | Old Category | Old Severity | Notes
+--------|--------------|--------------|--------------|--------------|-------
+CA3076 | Security | Info | Security | Hidden | DoNotUseInsecureXSLTScriptExecutionAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca3076)
```
Comment (mavasani) [inline src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md:5]: Can we avoid this severity change please?

File: src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/DoNotUseInsecureXSLTScriptExecution.cs [hunk 2]
```diff
@@ -1,283 +1,319 @@
...
-            isPortedFxCopRule: false,
-            isDataflowRule: false);
+
+        internal static DiagnosticDescriptor RuleDoNotUseInsecureXSLTScriptExecution =
+            DiagnosticDescriptorHelper.Create(
+                RuleId,
+                SecurityDiagnosticHelpers.GetLocalizableResourceString(nameof(MicrosoftNetFrameworkAnalyzersResources.InsecureXsltScriptProcessingMessage)),
+                SecurityDiagnosticHelpers.GetLocalizableResourceString(nameof(MicrosoftNetFrameworkAnalyzersResources.DoNotUseInsecureDtdProcessingGenericMessage)),
+                DiagnosticCategory.Security,
+                RuleLevel.IdeSuggestion,
```
Comment (dotpaul) [inline src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/DoNotUseInsecureXSLTScriptExecution.cs:26]:
```suggestion
                RuleLevel.IdeHidden_BulkConfigurable,
```

and undo changes to Analyzers.Unshipped.md.

File: src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md (hunk 1 above)
Comment (Evangelink) [reply src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md:5]: Sure!

**Review by mavasani — APPROVED** [review]
Thanks @Evangelink!

**Pull request merged**